.travis.yml

```diff
@@ -49,10 +60,8 @@ services:
  - memcache
  - redis-server
 
-before_install:
-  - sudo apt-get install -y axel cpanminus libtest-base-perl libtext-diff-perl liburi-perl libwww-perl libtest-longstring-perl liblist-moreutils-perl libgd-dev
-
 install:
+  - mkdir ~/bin && ln -s $(which ccache) ~/bin/$CC && export PATH=$HOME/bin:$PATH
```

Comment:
using "ln" is recommended way, it is documented in "man ccache", anyway, we save more on openssl, than nginx